src/main/java/com/google/summit/ast/expression/BinaryExpression.kt

```diff
@@ -47,7 +47,6 @@ class BinaryExpression(
     SUBTRACTION,
     MULTIPLICATION,
     DIVISION,
-    MODULO,
```

Comment:
just curious why was this here to begin with?

Reply:
I more-or-less used the grammar as the language spec. The grammar had a modulo operator, even though the language does not. This error was rectified in the latest version.